govcd/openapi_endpoints.go

```diff
@@ -0,0 +1,27 @@
+package govcd
+
+import "fmt"
+
+var endpointMinApiVersions = map[string]string{
+	"1.0.0/roles/": "31.0",
+}
+
+// checkOpenApiEndpointCompatibility checks if required VCD version is sufficient to work with this endpoint and returns
+// either error, either version
```

Comment:
Sorry, but a bit confused :) Maybe you could elaborate this comment a bit more and make some parts more specific:
- "required VCD version" - required by what?
- "either version" - which/who's version specifically?

In general, if you can just add more context, I'm sure it will be helpful in the future.

Thanks!!

Reply:
I hope I have improved it. This is the part of code we had discussions about. Will see if we stick with it after all reviews.